cheddar/cheddar-server/src/main/java/com/clicktravel/cheddar/server/runtime/config/RuntimeConfiguration.java

```diff
@@ -61,10 +61,11 @@
 
     public static boolean isLocalEnvironment(final Environment environment) {
         for (final String profile : environment.getActiveProfiles()) {
-            if (profile.equalsIgnoreCase(LOCAL_PROFILE)) {
+            if (profile.equalsIgnoreCase(LOCAL_PROFILE) || profile.equalsIgnoreCase(TEST_PROFILE)
```

Comment:
I stand by my original comment. Perhaps we take your method suggestion *in addition* to what is there, rather than replace it.